### ISSUE TITLE
export-installation --target vs import-installation --target arguments differ, causing confusion for users of om

### ISSUE THREAD

**Issue by dave-malone** [pivotal-cf/om#83]
When using om to test backup and restore of Ops Manager, we noticed that the export-installation command did not require us to prefix the Ops Man target value with https://, but when running import-installation, we had to prefix the value with https:// in order for the command to work. 

It would be nice to have consistency across these commands in order to improve usability. I can craft a PR for this later today.

Export command:
```
om --target $OPSMAN_HOST \
  --username $OPSMAN_USERNAME \
  --password $OPSMAN_PASS \
  --skip-ssl-validation \
export-installation \
  --output-file opsman-export.zip
```

Import command:
```
om --target https://$OPSMAN_HOST \
  --username $OPSMAN_USERNAME \
  --password $OPSMAN_PASS \
  --skip-ssl-validation \
import-installation \
  --installation opsman-export.zip \
  --decryption-passphrase $OPSMAN_DECRYPTION_PASSPHRASE
```

**Comment by davewalter** [comment]
Hi @dave-malone,

Which version of `om` are you using? It appears that this should have been fixed since v`0.21.0`: https://github.com/pivotal-cf/om/blob/master/network/unauthenticated_client.go#L46-L48

Regards,
Dave

**Comment by madamkiwi** [comment]
Thanks for the bringing this up @dave-malone !

It seems that [this line](https://github.com/pivotal-cf/om/blob/c1fc09813ed1601eefda0e60c7ab220a29301a09/network/unauthenticated_client.go#L41) is the issue, as it returns empty scheme and empty host for targets without the "https://" prefix, and then the scheme will be set to "https" but the host is now empty. [GoPlayground](https://play.golang.org/p/f3m7Y8jhm7)

Feel free to submit a PR and we'll take a look at it.
